### PR TITLE
Display service display name in License section

### PR DESCRIPTION
**Pull request by FredericEspiau**
## Display service display name in License section

Will display the service display name as long as the service id in the License section

### Applicable Issues

MANAGER-2462

### CODE REVIEW

File: client/app/license/license.controller.js [hunk 1]
```diff
@@ -1,127 +1,162 @@
...
-      WINDOWS: 'WINDOWS',
-    };
-    $scope.filterType = null;
-    $scope.$state = $state;
+import _ from 'lodash';
+
+angular
+  .module('Module.license')
+  .controller('LicenseCtrl', class {
+    constructor(
```
Comment (antleblanc) [inline client/app/license/license.controller.js]: You can specify the `/* @ngInject */` as well.

File: client/app/license/license.html [hunk 2]
```diff
@@ -57,6 +57,10 @@
                         property="serviceName"
                         type="text">
             </oui-column>
+            <oui-column title=":: 'license_dashboard_table_title_serviceDisplayName' | translate"
```
Comment (antleblanc) [inline client/app/license/license.html]: Missing some `data-` prefixes.

File: client/app/license/license.html [hunk 3]
```diff
@@ -71,35 +75,37 @@
...
             </oui-column>
 
-            <oui-action-menu align="end" compact>
+            <oui-action-menu
+                data-align="end"
+                data-compact>
                 <oui-action-menu-item
-                    text="{{:: 'license_dashboard_table_popover_details' | translate }}"
-                    href="{{ $state.href('app.license.detail', { licenseId: getLicenseId($row) }) }}">
+                    data-text="{{:: 'license_dashboard_table_popover_details' | translate }}"
```
Comment (antleblanc) [inline client/app/license/license.html]: According to the documentation the [`text`](https://ovh-ux.github.io/ovh-ui-kit-documentation/#!/oui-angular/action-menu#deprecated) attribut is now deprecated. Prefer using the transclude slot available.

File: client/app/license/license.html [hunk 4]
```diff
@@ -71,35 +71,37 @@
...
-                    href="{{ $state.href('app.license.upgrade', { licenseId: getLicenseId($row) }) }}">
+                    data-text="{{:: 'license_dashboard_table_popover_upgrade' | translate }}"
+                    data-href="{{ $state.href('app.license.upgrade', { licenseId: getLicenseId($row) }) }}">
                 </oui-action-menu-item>
                 <oui-action-menu-item
                     data-ng-if="$row.isExpired"
-                    text="{{:: 'license_dashboard_table_popover_renew' | translate }}"
-                    href="{{ getRenewUrl() + $row.id }}">
+                    data-text="{{:: 'license_dashboard_table_popover_renew' | translate }}"
+                    data-href="{{ getRenewUrl() + $row.id }}">
```
Comment (JeremyDec) [inline client/app/license/license.html]:
```suggestion
                    data-href="{{:: getRenewUrl() + $row.id }}">
```

**Comment by jleveugle** [comment]
Hello @FredericEspiau,
Your pull request is now closed due to inactivity.
Feel free to re-open it if you have any update.
Anna Bot for OVH UX <3

**Pull request closed without merging**